pipelinewise/cli/pipelinewise.py

```diff
@@ -1621,6 +1658,171 @@ def encrypt_string(self):
         print(yaml_text)
         print('Encryption successful')
 
+    def partial_sync_table(self):
+        """
+        Partial Sync Table
+        """
+
+        cons_target_config = None
+
+        # Continue only if tap and target is supported by partial sync
+        try:
+            self._check_supporting_tap_and_target_for_partial_sync()
+
+            tap_id = self.tap['id']
+            tap_type = self.tap['type']
+            target_id = self.target['id']
+            target_type = self.target['type']
+            fastsync_bin = utils.get_fastsync_bin(self.venv_dir, tap_type, target_type)
```

Comment:
`get_fastsync_bin` in partial sync ?

Reply:
They use same method for getting the bin. I rename the variable.

- [ ] rename `fastsync_bin` variable